openedx/features/data_extract/helpers.py

```diff
@@ -78,7 +78,13 @@ def get_user_demographic_data(profile):
                              data={'_uid': 1, 'username': profile.user.username},
                              headers=headers)
 
-    user_community_data = json.loads(response._content)['payload']
+    try:
+        user_community_data = json.loads(response._content)['payload']
+        reputation = user_community_data['reputation']
```

Comment:
you can use user_community_data.get('reputation', 0) to skip exception handling

Reply:
done